hexgame/player.py

```diff
@@ -21,37 +21,31 @@ def __init__(self, color: Color = Color.Red, mode: PlayerMode = PlayerMode.AI):
     def __eq__(self, other: object) -> bool:
         if not isinstance(other, Player):
             return False
-
         return self.color == other.color
 
-    def _place_stone(self, board: Board, i, j):
-        board.place_stone(i, j, self.color)
-
     # here we define the playing logic
-    def play(self, board: Board):
+    def play(self, board: Board) -> tuple[int, int, Color]:
```

Comment:
i like added types

Reply:
also, i think honestly this is fine like this but if we want to be extra fancy we could have turned this composite type tuple[int,int,color] into its own type, Move.

Reply:
Yeah, would be nicer, I agree, but I also think it would be out of the scope of this PR, can be done in a future refactoring PR that will have that objective exclusively.